### PR TITLE
Init_model

### PR DESCRIPTION
**Pull request by hieuph-0968**
Screenshot from 2019-11-26 17-04-56

### CODE REVIEW

**Comment by hieuph-0968** [comment]
Ready

**Comment by hieuph-0968** [comment]
Ready

File: app/models/user.rb [hunk 1]
```diff
@@ -0,0 +1,17 @@
+class User < ApplicationRecord
+  has_many :posts, dependent: :destroy
+  has_many :comments, dependent: :destroy
+  has_many :likes, dependent: :destroy
+  has_many :user_histories, dependent: :destroy
+  has_many :active_messages, class_name: "Message",
```
Comment (huyhm-0966) [inline app/models/user.rb]: Message.name thay cho "Message"

File: app/models/user.rb [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+                             foreign_key: :messager_id,
+                             dependent: :destroy,
+                             inverse_of: :user
+  has_many :passive_messages, class_name: "Message",
```
Comment (huyhm-0966) [inline app/models/user.rb]: Message.name thay cho "Message"

**Comment by huyhm-0966** [comment]
merged
Có cmt em fix pull sau nhé

**Pull request merged**